common/src/commonMain/kotlin/io/xorum/codeforceswatcher/util/IFeedbackController.kt

```diff
@@ -0,0 +1,78 @@
+package io.xorum.codeforceswatcher.util
+
+abstract class IFeedbackController {
```

Comment:
The convention is:
 - when you have `interface`, you start its name with `I` or make it an `*able` (`Flyable`, `Movable`, etc).
 - when you have `abstract class` you use `Base` prefix (in this case `BaseFeedbackController`).